pyanaconda/modules/common/constants/objects.py

```diff
@@ -122,11 +122,11 @@
 )
 
 PAYLOAD_LIVE_IMAGE = DBusObjectIdentifier(
-    namespace=PAYLOAD_NAMESPACE,
+    namespace=PAYLOADS_NAMESPACE,
     basename="LiveImage"
 )
 
 PAYLOAD_LIVE_OS = DBusObjectIdentifier(
-    namespace=PAYLOAD_NAMESPACE,
+    namespace=PAYLOADS_NAMESPACE,
```

Comment:
Payloads should use the namespace `PAYLOAD_BASE_NAMESPACE`. Then, for example, the DBus name of `PAYLOAD_LIVE_OS` will be `.Payloads.Payload.LiveOS`. Is this change planned for the pull request that will introduce dynamic payload modules?

Reply:
Yes, that will be introduced by the dynamic payloads PR.